terraform/application/config/production.tfvars.json

```diff
@@ -0,0 +1,11 @@
+{
+    "cluster": "production",
+    "namespace": "tra-production",
+    "evidence_container_retention_in_days": 30,
+    "enable_monitoring": true,
```

Comment:
Can we keep it false for the time being? We'll enable it closer to migration day